packages/web-api-scan-request-sender/src/sender/on-demand-dispatcher.ts

```diff
@@ -32,9 +32,11 @@ export class OnDemandDispatcher {
         let continuationToken;
 
         do {
```

Comment:
Do we need this outer loop after this change? 
we are going to scan totalItemsToAdd (or less if the query returned less) all at once in the inner loop or it will return no items

Reply:
as discussed offline, we dont need both the loops. but that needs to be fixed once we refactor the cosmoscontainerclient.readItems. not taking that as part of this pr.